app/services/update_plan_service.rb

```diff
@@ -420,8 +431,9 @@ def send_order(order_id)
     end
     work_order = WorkOrder.find(order_id)
 
-    if (work_order.jobs.size == 0)
-      work_order = work_order_splitter.split(work_order)
+    if work_order.jobs.size == 0 && !work_order_splitter.split(work_order)
+      add_error("The work order could not be split into jobs.")
+      return false
     end
```

Comment:
I may be wrong... but I think that if `work_order` gets split, it will need a reload here afterwards before it gets dispatched.

Reply:
Oops, apologies. I meant after it gets split, the reference to the `work_order` in the `update_plan_service` needs reloading before it gets passed to the WorkOrderDispatcher.

I think otherwise it wouldn't know it has any new `Jobs`.